packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1803,6 +1803,26 @@ For brevity, the following response doesn't include all available properties.
         "description": "Configuration properties specific to Splunk Cloud",
         "type": "object",
         "properties": {
+          "edition": {
+            "title": "Splunk Edition",
```

Comment:
```suggestion
            "title": "Splunk Cloud instance edition",
```
I'm not sure if this is the official title of this feature? We normally capitalize the official feature name. Can you verify what Splunk uses?

Reply:
We still haven't heard back from Splunk re naming (https://okta.slack.com/archives/CPR1T3GG7/p1670617425367029?thread_ts=1670362962.551249&cid=CPR1T3GG7) and the title is defining form field name, I'd leave it as is for now